core/src/main/java/org/hisp/dhis/android/core/user/internal/LogInCall.kt

```diff
@@ -182,7 +172,7 @@ internal class LogInCall @Inject internal constructor(
 
         var credentials: Credentials? = null
         return try {
-            val user = apiCallExecutor.executeObjectCallWithErrorCatcher(authenticateCall, apiCallErrorCatcher)
+            val user = coroutineAPICallExecutor.wrap { authenticateCall }.getOrThrow()
```

Comment:
The `authenticateCall`is executed at the time it is declared. Before, it was a `Callable` so it was executed when its method `call` was invoked; but now it is a suspend function.
The actual invokation must be wrapped by the `wrap` function.